dist/ti-modal.js

```diff
@@ -108,6 +110,20 @@ define("ti-modal",
     });
 
     ModalComponent = Ember.Component.extend({
+      bindClosingBehavior: function() {
+        var modalComponent = this;
+        
+        $(document).bind('keyup.modal', function(e) {
```

Comment:
`As of jQuery 1.7, the .on() method is the preferred method for attaching event handlers to a document.` from jq docs.

You could also use `$.proxy` to keep the `this` scope and avoid having to define `modalComponent` entirely.